### PR TITLE
Grey & Wooden Railings

### PR DESCRIPTION
**Pull request by rosieposieeee**
## About the PR

I like these modern railings, but the bright white isn't perfect for a lot of situations. Added wooden and grey ones.

Content Client_9L2UQ8ITve

Content Client_75fndxMU27

The wooden ones even break into wood. :)

NOTE: I hereby release all changes made in this PR under MIT license.

## Requirements

- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl:
- add: Added wooden and grey modern railings.

### CODE REVIEW

File: Resources/Prototypes/DeltaV/Entities/Structures/Walls/railing.yml [hunk 1]
```diff
@@ -37,3 +37,195 @@
...
+          collection: WoodDestroy
+          params:
+            volume: -6
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          MaterialWoodPlank1:
+            min: 0
+            max: 1
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Structures/Walls/railing.yml:117]: put this into a `BaseRailingWood` abstract prototype and add it to the wood railings parents

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/DeltaV/Entities/Structures/Walls/railing.yml (hunk 1 above)
Comment (rosieposieeee) [reply Resources/Prototypes/DeltaV/Entities/Structures/Walls/railing.yml:117]: oh shit right we can multi parent

File: Resources/Prototypes/DeltaV/Entities/Structures/Walls/railing.yml [hunk 2]
```diff
@@ -37,3 +37,127 @@
...
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: round_grey
+  - type: Icon
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: round_grey
+
+- type: entity
+  id: BaseRailingWood
+  suffix: Modern, Wood
+  components:
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Structures/Walls/railing.yml]: add `abstract: true` above id so this cant be spawned directly

File: Resources/Prototypes/DeltaV/Entities/Structures/Walls/railing.yml [hunk 3]
```diff
@@ -37,3 +37,127 @@
...
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: corner_small_wood
+  - type: Icon
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: corner_small_wood
+
+- type: entity
+  parent: [BaseRailingWood, RailingRound]
+  id: RailingRoundDeltaVWood
+  suffix: Modern
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Structures/Walls/railing.yml]: remove suffix for all these, BaseRailingWood has it

File: Resources/Prototypes/DeltaV/Entities/Structures/Walls/railing.yml [hunk 4]
```diff
@@ -37,3 +37,127 @@
...
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: round_wood
+  - type: Icon
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: round_wood
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Structures/Walls/railing.yml]: newline at end of file

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/DeltaV/Entities/Structures/Walls/railing.yml (hunk 2 above)
Comment (rosieposieeee) [reply Resources/Prototypes/DeltaV/Entities/Structures/Walls/railing.yml]: 👍

File: Resources/Prototypes/DeltaV/Entities/Structures/Walls/railing.yml (hunk 3 above)
Comment (rosieposieeee) [reply Resources/Prototypes/DeltaV/Entities/Structures/Walls/railing.yml]: oops i did it for two but missed the other two👍

File: Resources/Prototypes/DeltaV/Entities/Structures/Walls/railing.yml (hunk 4 above)
Comment (rosieposieeee) [reply Resources/Prototypes/DeltaV/Entities/Structures/Walls/railing.yml]: 👍

Comment (rosieposieeee) [reply Resources/Prototypes/DeltaV/Entities/Structures/Walls/railing.yml]: 👍

**Review by deltanedas — APPROVED** [review]

**Pull request merged**